Sources/LiCore/Networking/LiClient.swift

```diff
@@ -288,7 +288,7 @@ extension LiClient {
         case .liUploadImageClient:
             return [:]
         case .liRepliesClient(let requestParams):
-            return [LiQueryConstant.liMarkAsRead: "true", "q": LiUrlConstructor.getLiqlQuery(client: self).replacingOccurrences(of: "##", with: requestParams.parentId)]
+            return [LiQueryConstant.liMarkAsRead: "true", "q": LiUrlConstructor.getLiqlQuery(client: self).replacingOccurrences(of: "##", with: requestParams.parentId).replacingOccurrences(of: "**", with: "\(requestParams.limit)").replacingOccurrences(of: "&&", with: "\(requestParams.offset)")]
```

Comment:
are you sure the other params will not contain these chars (** ..etc)?

Reply:
Yes

Reply:
Also, in the future instead of using stringByReplacingString check out string formatter.

Reply:
aye aye